docs/webhooks/01-Overview.md

```diff
@@ -280,7 +290,7 @@ Sent when an incident is newly created/triggered.
 
 ### incident.unacknowledged
 
-`data.type` is [`incident`](#incident)
+`data.type` is [`incident`](##incident)
```

Comment:
Is this correct? I think this breaks the link.

Reply:
typo.

Reply:
I removed it in the next PR.